### PR TITLE
exclude summative assessments from appearing in results by item and d…

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
…istractor analysis

### CODE REVIEW

File: webapp/src/main/resources/application.sql.yml [hunk 1]
```diff
@@ -560,7 +560,9 @@ sql:
           i.options_count,
           i.type
         from item i
-        where i.asmt_id=:assessment_id
+          join asmt a on i.asmt_id=a.id
+        where a.type_id != 3
```
Comment (dsnyder-fairway) [inline webapp/src/main/resources/application.sql.yml]: type_id of 3 is a summative assessment. I'm open to either passing the summative assessment types to include/exclude from the methods that call this as well. @esotrope, thoughts?

Comment (esotrope) [reply webapp/src/main/resources/application.sql.yml]: 🤔  An include-style query would be more future-proof if other assessment types are added

Comment (esotrope) [reply webapp/src/main/resources/application.sql.yml]: Oh - If we are selecting an assessment by ID I don't think we need the type filter right?
I don' think there will be like an iab{id:1} and a summative{id:1}.

Comment (dsnyder-fairway) [reply webapp/src/main/resources/application.sql.yml]: type is used for item type (MC, MS, etc)

**Comment by dsnyder-fairway** [comment]
@jtreuting this change prevents viewing Summative test results via the Distractor Analysis or Results by Item

**Pull request merged**